src/main.cpp

```diff
@@ -342,6 +342,8 @@ int main ( int argc, char** argv )
         // HTML status file ----------------------------------------------------
         if ( GetStringArgument ( argc, argv, i, "-m", "--htmlstatus", strArgument ) )
         {
+            qWarning() << qUtf8Printable (
+                QString ( "- The HTML status file option (-m) is deprecated and will be removed soon. Use JSON-RPC instead." ) );
```

Comment:
I think it's best to use the long option in the message or
```suggestion
                QString ( "- The HTML status file option (\"--htmlstatus\" or \"-m\") is deprecated and will be removed soon. Please se JSON-RPC instead.  See <url to spec>." ) );
```
This implies the JSON-RPC interface will be stablised and formalised.

Reply:
```suggestion
                QString ( "- The HTML status file option (\"--htmlstatus\" or \"-m\") is deprecated and will be removed soon. Please use JSON-RPC instead.  See https://github.com/jamulussoftware/jamulus/blob/main/docs/JSON-RPC.md." ) );
```

Reply:
Mmm, OK but I'm going to put a "needs documentation" on this.  We should have something better than in the Github source.  It needs to be on the website, I think.

Reply:
I think it's already labelled deprecated for a while now. But the next step is tracked in #3362 already.

Reply:
OK, now it's linked here, I'm happy 👍🏻